### ISSUE TITLE
[CSS]: Css auto-completion doesn't have several values of 'display'

### ISSUE THREAD

**Issue by Alexey-T** [sublimehq/Packages#3039]
### What happened?

Proof: https://developer.mozilla.org/en-US/docs/Web/CSS/display

Paste this text, I marked few items which are NOT suggested by auto-complete, by `MISS`.
```
p{
/* legacy values */
display: block;
display: inline;
display: inline-block;
display: flex;
display: inline-flex;
display: grid;
display: inline-grid;
display: flow-root; /* MISS */

/* box generation */
display: none;
display: contents;

/* two-value syntax */
display: flow; /* MISS */
display: inline flow;
display: inline flow-root;
display: block flex;
display: inline flex;
display: block grid;
display: inline grid;
display: block flow-root;

/* other values */
display: table;
display: table-row; /* all table elements have an equivalent CSS display value */
display: list-item;

/* Global values */
display: inherit;
display: initial; 
display: revert; /* MISS */
display: unset;
}
```

**Comment by deathaxe** [comment]
Auto completion is far a way from being up to date at all. It touches nearly everything from properties, property values etc...

That's a known issue.

An attempt to improve that was https://github.com/sublimehq/Packages/pull/2737

It involves quite a lot manual work to convert those mdn rules and add probably missing stuff from other sources.

It furthermore brings current completion plugin's architecture to it's limits due to those huge data files.

**Comment by Alexey-T** [comment]
So let's apply the #2737 and test my report after it?

**Comment by deathaxe** [comment]
CSS has moved on since then. Also https://github.com/sublimehq/Packages/pull/2968 interferes with that change and needs to be merged first. Once done another roundtrip to update the PR was needed.

It also adds many builtin variables to sync syntax and completions, but it slows down CSS syntax significantly. That's the main reason for droping that PR next to the time expensive task fo syncing those lists with MDN while not breaking any existing stuff.

CSS completions would also need better support for at-rules and selectors. All of that rises the question, whether this plugin would grow beyound the scope of this repo then.

**Comment by Alexey-T** [comment]
Okay, let’s postpone this issue for future.
